pages/login_page.rb

```diff
@@ -13,7 +13,7 @@ class LoginPage < Page
     def initialize url:, driver:, wait:
         super
         puts "navigating to login page if not here"
-        @driver.get @url
+        go_to @url
```

Comment:
Possibly could be re-named something like "driver_navigate." Naming it something simple could be a problem if we end up using some other library that has a `go_to` method